index/templates/all-petitions.html

```diff
@@ -0,0 +1,96 @@
+{% extends "base.html" %}
+{% block title %}View All Petitions{% endblock %}
+
+{% block content %}
+
+<div class="container">
+    <h3 class="text-dark headingbar pontiac">All Petitions</h3>
+    <div class="row">
+        <div class="col-sm-5">
+            <div class="checkbox">
+                <label>
+                    <input type="checkbox" aria-label="All resolved petitions checkbox"> Show All Resolved
+                </label>
+            </div>
+        </div>
+        <div class="col-sm-4">
+            <div class="checkbox">
+                <label>
+                    <input type="checkbox"> Show All Recognized
+                </label>
+            </div>
+        </div>
+        <div class="col-sm-3">
+            <div class="dropdown float-right">
+                <button class="btn btn-secondary dropdown-toggle" type="button" id="tagdropdown" data-toggle="dropdown"
+                    aria-haspopup="true" aria-expanded="false">
+                    Sort By:
+                </button>
+                <div class="dropdown-menu" aria-labelledby="tagdropdown">
+                    <a class="dropdown-item" href="#">Most Recent</a>
+                    <a class="dropdown-item" href="#">Most Signatures</a>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<br>
+
+<div class="container">
+    <ul class="card-columns">
+        {% for petition in petitions %}
+        <li class="card bg-light">
+            <div class="card-body text-center">
+                <h5 class="card-title text-center"><a href="#">{{ petition.title }}</a></h5>
+                <h6 class="card-subtitle mb-2">{{ petition.signatures.count }} signatures</h6>
+                    {% for tag in petition.tags.all %}
+                        <a href="#" class="link">{{ tag }}</a><br>
+            </div>
+        </li>
+        {% endfor %}
+        <!--If there are no petitions, display this message-->
+        {% empty %}
+            <p>No petitions yet. Stay tuned!</p>
+        {% endfor %}
+    </ul>
+</div>
+
+<!--Paginator Test-->
+{% if petitions.has_other_pages %}
+    <ul class="pagination justify-content-center">
+        <!--if there are any accessable petitions before the page that we're on, be able to get access to that 'page' of information-->
+        {% if petitions.has_previous %}
+            <li class="page-item">
+                <a class="page-link" href="?page={{ petitions.start_index }}">First</a>

```

Comment:
It looks like this link isn't working. Not quite sure why, because it seems like the last page button is working.

Also, we might not need first and last buttons, since there is a button for each page, but on smaller screen sizes it would be useful. Let me know what you think about this.

Reply:
Will work on that in the near future

Reply:
It works now. Will be included in the next pull request